recaptcha_enterprise/snippets/passwordLeakAssessment.ts

```diff
@@ -0,0 +1,220 @@
+/**
```

Comment:
Same comments as the js file. Why are there 2 files? was the js one commited by mistake?

Reply:
No, it's WAI.
We do not have a separate repository for typescript. Hence all ts code will be compiled to Javascript and the tests will be executed.